devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -435,6 +436,19 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
         }
     }
 
+     if (!(config.check("CalibrationConfig") && config.find("CalibrationConfig").isList())) {
+        yInfo() << LogPrefix << "CalibrationConfig option not found or not valid, all the model joints are selected.";
```

Comment:
This info message should be updated. If the option is not valid, we are using the "zero configuration"